tests/xml2/XMLBearTest.py

```diff
@@ -36,6 +37,10 @@ def load_testdata(filename):
 <a>hey and hi</a>
 """
 
+invalid_xml_file_c14n = """<?xml version="1.0"?>
```

Comment:
It's not clear without knowing the c14n style, why this is invalid. Could you please add a comment explaining why this fails under this style?